lnwallet/channel_test.go

```diff
@@ -111,7 +111,7 @@ func testAddSettleWorkflow(t *testing.T, tweakless bool,
 	// we expect the messages to be ordered, Bob will receive the HTLC we
 	// just sent before he receives this signature, so the signature will
 	// cover the HTLC.
-	aliceNewCommit, err := aliceChannel.SignNextCommitment()
+	aliceNewCommit, err := aliceChannel.SignNextCommitment(nil)
```

Comment:
An alternative approach would be to add `WithQuitChannel` to `*LightningChannel`, then use that. Then need to make sure we pass a parent quit chan into every call to `NewLightningChannel()`. So not sure if that ends up resulting in a smaller or larger diff...

Reply:
I had the same thought, but AFAICT that would give us a way larger diff without a functional difference.